prow/config.yaml

```diff
@@ -6455,6 +6455,27 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  kubernetes-incubator/service-catalog:
+  - name: svc-cat-echo
+    agent: kubernetes
+    context: svc-cat-echo
+    always_run: true
+    rerun_command: "/test svc-cat-echo"
+    trigger: "(?m)^/test (all|svc-cat-echo)\\s*"
+    decorate: true
+    spec:
+      containers:
+      - image: alpine
+        command:
```

Comment:
pretty sure this should be:
```yaml
command:
- sh
args:
- -c
- env; ls; echo "The artifacts dir is ${ARTIFACTS}"
```

Reply:
okay, not sure what the rules are, trying to figure it out.
this one was riffing off of the job
```
tensorflow/minigo: 
 - name: pull-tf-minigo-cc
```
which has 
```
        command:
        - /bin/bash
        - -c
```
but I do also see now that it has the args as `&&` together of commands.

Reply:
note to self: go try and read the prow code to see what the prow spec is transformed into, i.e. a podspec or containerspec, etc.

Reply:
It's a PodSpec

Reply:
yes, exactly so:
https://github.com/kubernetes/test-infra/blob/1bb0838c9b49beb86d4c25a795a0a2f3ec317f0c/prow/kube/prowjob.go#L145-L147

Reply:
who is specifying the containers `Name` field?